src/main/java/ch/njol/skript/expressions/ExprCommand.java

```diff
@@ -78,19 +80,21 @@ public boolean init(final Expression<?>[] exprs, final int matchedPattern, final
 			s = ((PlayerCommandPreprocessEvent) e).getMessage().substring(1).trim();
 		} else if (e instanceof ServerCommandEvent) {
 			s = ((ServerCommandEvent) e).getCommand().trim();
+		} else if (Skript.classExists("org.bukkit.event.command.UnknownCommandEvent") && e instanceof UnknownCommandEvent) {

```

Comment:
instead of checking this each time, store it in a final static field like `UNKNOWN_COMMAND_SUPPORTED` and use that field